components/maskedtextbox/overview.md

```diff
@@ -92,7 +91,7 @@ The following parameters enable you to customize the [appearance]({%slug maskedt
 
 ## MaskedTextBox Reference and Methods
 
-The MaskedTextBox proves a `FocusAsync` method that enables programmatic focus. To use it, obtain reference to the component instance.
+The MaskedTextBox proves a `FocusAsync` method that enables programmatic focus. To use it, obtain reference to the component instance via `#ref`. @[template](/_contentTemplates/common/inputs.md#focus-kb)
```

Comment:
This sentence appears in the rest of the `overview.md` files, I suggest changing it there as well.
```suggestion
The MaskedTextBox proves a `FocusAsync` method that enables programmatic focus. To use it, obtain a reference to the component instance through `#ref`. @[template](/_contentTemplates/common/inputs.md#focus-kb)
```